### ISSUE TITLE
Mathstats requirement

### ISSUE THREAD

**Issue by amandaamelo** [ksahlin/BESST#86]
Hey there! 

I'm trying to use BESST for the first time. It is installed locally and globally on my Linux environment. When I call the program I have the usage information returned so it is installed fine.

However, when I insert my arguments, I get the message that mathstats is not installed:
Uninstalled module: mathstats
Please install mathstats using one of the two provided commands from terminal:
         pip install mathstats
         easy_install mathstats

However, it is already installed:
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: mathstats in /usr/local/lib/python3.8/dist-packages (0.2.6.5)
Requirement already satisfied: scipy>=0.9 in /usr/lib/python3/dist-packages (from mathstats) (1.3.3)

I have tried everything and cannot seem to make BESST run. Any help would be appreciated! Thanks

**Comment by ksahlin** [comment]
Hi @amandaamelo, 

>  It is installed locally and globally on my Linux environment. 

Perhaps this is the issue. One of your installations does not install `mathstats` because its already installed (pip finds it elsewhere), but when you run the BESST, the program itself expects the library to be avaialble in the path where the installation is. 

Solution _could be_ to uninstall both installations completely (removing also mathstats) and then install BESST _once_ from scratch.

**Comment by amandaamelo** [comment]
Hi ksahlin! Thanks for the response!

I tried this and still did not work. It only worked when I installed a virtual environment on my server and through that, it worked.
I also had to correct a python version issue of one of the scripts inside the virtual environment (https://github.com/jupyter/nbviewer/issues/869) before making it work.

Anyways, probably a configuration issue from my server, since I did not see another person with this problem. Thanks anyway!
